core/models/project.py

```diff
@@ -61,9 +61,7 @@ class AppMeta:
                                    null=True,
                                    help_text='Lay summary should provide a brief overview of project goals and approach. Lay summary may be displayed publicly if the project\'s data gets published in the data catalog')
 
-    dpia = models.TextField(verbose_name='DPIA Link',
```

Comment:
While this field is removed from the model, the migration script does not contain any corresponding change to the database. Are you sure you ran makemigrations after removing this field?

Reply:
Fixed it. thank you